nucypher/network/nodes.py

```diff
@@ -749,7 +751,7 @@ def learn_from_teacher_node(self, eager=False, canceller=None):
 
         if not self.done_seeding:
             try:
-                remembered_seednodes = self.load_seednodes(record_fleet_state=False)
+                remembered_seednodes = self.load_seednodes(record_fleet_state=True)
```

Comment:
What's the implication of changing here this from False to True?

Reply:
Previously nodes remembered in `known_nodes` were accessible right away after they were recorded. This PR makes it more strict - only those nodes that make up the current fleet state are accessible, and the newly recorded nodes are in temporary storage. So in order for the seednodes to be visible we must record the fleet state.